### DISCUSSION TITLE
Plan to support parallel controller ?

### DISCUSSION THREAD

**Discussion by leon-qr-li** [anasoid/jmeter-as-code#179] [Q&A] [answered]
Hi, I am wondering do we support parallel controller or are we planning to support it in near future ? 

Our test plan will heavily based on bzm-parallel controller plugin. And it is a popular plugin from my understand, it will be great if we can add this controller type in near future.

**Comment by anasoid** [comment, marked as answer]
Hello,
I plan to add most common used plugins.

If you have a small use case, and yo need it quickly you cane use 'include' and you can have maxthread as parameter  see https://jmc.anasoid.org/docs/users/include.

If you need other components create an issue for each component i will add them in 2 or 3 weeks

**Reply by leon-qr-li** [reply]
Thank you for the quick feedback and appreciate the work ! Feel free to let me know if you need contributor for the library, I will see if I can help. 
We like this library a lot ! :) we were trying to build our own "jmeter as code" framework until I see this one. Can't wait to see more popular plugin added in near future. For now, it would be great if "Parallel controller" can be added next.

**Reply by anasoid** [reply]
I will add it in 2 or 3 weeks :D

**Reply by anasoid** [reply]
a new  milestone is scheduled https://github.com/anasoid/jmeter-as-code/milestone/7l

**Reply by anasoid** [reply]
A new release V0.6.5 with Parallel controller

https://jmc.anasoid.org/docs/reference/jmeter-plugins/jmeter-plugins-component#parallel-controller--sampler
